plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -83,8 +88,52 @@ export const CatalogTable: FC<CatalogTableProps> = ({
     );
   }
 
+  const actions: TableProps<Entity>['actions'] = [
+    (rowData: Entity) => {
+      const location = findLocationForEntityMeta(rowData.metadata);
+      return {
+        icon: GitHub,
+        tooltip: 'View on GitHub',
+        onClick: () => {
+          if (!location) return;
+          window.open(location.target, '_blank');
+        },
+        hidden: location?.type !== 'github',
+      };
+    },
+    (rowData: Entity) => {
+      const createEditLink = (location: LocationSpec): string => {
+        switch (location.type) {
+          case 'github':
+            return location.target.replace('/blob/', '/edit/');
+          default:
+            return location.target;
+        }
+      };
+      const location = findLocationForEntityMeta(rowData.metadata);
+      return {
+        icon: Edit,
+        tooltip: 'Edit',
+        // iconProps: { size: 'small' },
+        onClick: () => {
+          if (!location) return;
+          window.open(createEditLink(location), '_blank');
+        },
+        hidden: location?.type !== 'github',
+      };
+    },
+    (rowData: Entity) => {
+      const isStarred = isStarredEntity(rowData);
+      return {
+        icon: isStarred ? () => <Star htmlColor="#f3ba37" /> : StarOutline,
```

Comment:
Is this colour in the `Theme` anywhere?

Reply:
Don't think so. The reason it's here is that I tried to copy over alunds previous code for the star that used a withStyles (and that hardcoded color too), but that broke types at his line that I was unable to fix at the time. So I switched to using the real htmlColor prop instead and went on with my day :)